src/views/ThreeWActivityDetail/ActivityListItem/index.tsx

```diff
@@ -0,0 +1,153 @@
+        peopleHouseholds,
+        title,
+        sectorDetails,
+        customAction,
+        customSupply,
+        activitySupply,
+        activityDetails,
+        isSimplifiedReport,
+    } = props;
+    const strings = useTranslation(i18n);
+
+    const {
+        response: optionsResponse,
+    } = useRequest({
+        url: '/api/v2/emergency-project/options/',
+    });
+
+    const supplyIdToTitleMap = useMemo(() => (
+        listToMap(
+            optionsResponse?.actions?.flatMap(
+                (detail) => detail.supplies_details,
+            ).filter(isDefined),
+            ((detail) => detail.id),
+            ((detail) => detail.title),
+        )
+    ), [optionsResponse?.actions]);
+
+    return (
+        <Container
+            className={styles.activityList}
+            heading={title ?? customAction}
+            headingLevel={3}
+            childrenContainerClassName={styles.actionContent}
+        >
+            <TextOutput
+                label={strings.emergencySector}
+                value={sectorDetails}
+                strongLabel
+            />
+            <div>
+                {activityDetails}
+            </div>
+            {isSimplifiedReport && peopleHouseholds === 'people' && (
+                <>
+                    {(isDefined(maleCount)
+                        || isDefined(femaleCount))
+                        && (
+                            <>
+                                <TextOutput
+                                    label={strings.emergencyMale}
+                                    value={maleCount}
+                                    strongLabel
+                                />
+                                <TextOutput
+                                    label={strings.emergencyFemale}
+                                    value={femaleCount}
+                                    strongLabel
+                                />
+                            </>
+                        )}
+                    <TextOutput
+                        label={strings.emergencyTotalPeople}
+                        value={peopleCount}
+                        strongLabel
+                    />
+                </>
+            )}
+            {peopleHouseholds === 'households' && (
+                <TextOutput
+                    label={strings.emergencyTotalHouseholds}
+                    value={householdCount}
+                    strongLabel
+                />
+            )}
+            {activitySupply && Object.keys(activitySupply).length > 0 && (
+                <Container childrenContainerClassName={styles.supplyContent}>
+                    <div className={styles.supplyHeader}>
+                        {strings.emergencySupplies}
+                    </div>
+                    {activitySupply
+                        && (Object.keys(
+                            activitySupply,
+                        ) as unknown as number[]).map((supply) => (
```

Comment:
Casting the value of Object.keys as number[] is not correct. This will always be string[]